content/faq/shapeshift.md

```diff
@@ -0,0 +1,44 @@
+---
+title: How can I convert my crypto into LBC?
+category: wallet
+---
+
+The ability to convert your cryptoassets into LBRY Credits (LBC) is available directly within the LBRY app, powered by [ShapeShift](https://info.shapeshift.io/about). LBRY Inc. does not handle any of the incoming funds and the LBC are sent from ShapeShift directly. 
+
+*Note: ShapeShift is unavailable in New York and Washington (and possibly surrounding areas if your ISP's goelocation is not accurate). Please see [ShapeShift](https://shapeshift.io).* 	 
```

Comment:
The `Please see ShapeShift` seems out of place to me. Maybe `Please see ShapeShift for more info on this` or something like that. Do they have a page about this specific issue on their site?

Reply:
If you are blocked and you go there, you get a big warning